### DISCUSSION TITLE
What is _encryption.py::AlgV5.generate_values good for?

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/pypdf#1146] [Q&A] [answered]
I see that nothing calls [`_encryption.py::AlgV5.generate_values`](https://github.com/py-pdf/PyPDF2/blob/main/PyPDF2/_encryption.py#L566-L578). Does anybody know what it's good for?

**Comment by MartinThoma** [comment, marked as answer]
@exiledkingcc You've introduced it with https://github.com/py-pdf/PyPDF2/pull/1015 . Do you remember why you've added that function?

**Reply by exiledkingcc** [reply]
when encrypting pdf using algorithm v5, the OE OU should be added to Encrypt enctry.
i was planning to  rewrite the encryption part after finishing the decryption part done, but these days i'm too busy. sorry for that.

**Reply by MartinThoma** [reply]
No problem - we are all (or at least most of us, I guess) working in full-time jobs. We know the feeling :hugs: 

I'll leave it as-is and hope you can pick up on it at some point :-)
